pkg/querybuilder/fallback_expr.go

```diff
@@ -116,3 +120,58 @@ func GroupByKeys(keys []qbtypes.GroupByKey) []string {
 	}
 	return k
 }
+
+func FormatValueForContains(value any) string {
```

Comment:
would be great if there is independent testcase for this.

Reply:
Added in [f2a648e](https://github.com/SigNoz/signoz/pull/8768/commits/f2a648e9a69155a01bbdc8a422364499f9cc5879)